static/js/avatar.js

```diff
@@ -21,13 +21,16 @@ export function build_bot_create_widget() {
     const $input_error = $("#bot_avatar_file_input_error");
```

Comment:
You need to line-wrap paragraphs within commit messages:

![image](https://user-images.githubusercontent.com/2746074/216684796-d7c750ab-3ca7-4152-88d2-a3650afe0b1c.png)

Most editors have a feature to do this.